src/app/components/challenge/challengesubmissions/challengesubmissions.component.ts

```diff
@@ -134,6 +143,10 @@ export class ChallengesubmissionsComponent implements OnInit, AfterViewInit {
    */
   apiCall: any;
 
+  columnsToDisplay = ['status', 'stderr_file', 'stdout_file', 'submission_result_file'];
```

Comment:
On the frontend, display them after capitalization and spacing. That is, column names (to be rendered): 'Status', 'Stderr File', 'Stdout File', 'Submission Result File'.

Reply:
Done